GWAS/LMM.ipynb

```diff
@@ -270,9 +270,9 @@
     "# Summary statisticss format file path used for unifying output column names. Will not unify names if empty\n",
     "parameter: formatFile = path('.')\n",
     "# Qualitative covariates to be used in the analysis\n",
-    "parameter: covarCol = []\n",
+    "parameter: covarCol = list\n",
```

Comment:
@dianacornejo I changed it back to `[]` becuase when you require `list` it means covarCol is required. But if you input `covarCol = []` it means covarCol is optional with default to empty. So please change it back to `[]`, unless am missing something?